src/components/canvas-tools/image-tool.sass

```diff
@@ -85,6 +85,6 @@
       visibility: visible
 
   .image-tool-image
-    background-repeat: no-repeat
+    background-repeat: no-repeat !important
```

Comment:
Why is this necessary?

Reply:
It seems that updating the background image via the background style can cause styles set in css to be ignored or overwritten - images in the sidebar would suddenly come up repeated. I'm adjusting the call so that all styling happens in one place (the component, since dimensions are dynamic) for single source of authority.

Reply:
I think if you used `background-image` instead of `background` in the TypeScript, then it probably wouldn't override the `background-repeat` in CSS, but handling it all in TypeScript should work, too.